### DISCUSSION TITLE
Mass editing on the weblate.

### DISCUSSION THREAD

**Discussion by antoniospneto** [OCA/odoo-community.org#118] [Q&A] [answered]
Does anyone know if there is any way to edit in mass or perform an automated action on the weblate?

For example, in Brazil we have registered 5 thousand [cities](https://github.com/OCA/l10n-brazil/blob/14.0/l10n_br_base/data/res.city.csv) and the name of these cities does not require translation, but they are pending in the weblate, adjusting them one by one manually through the weblate's web interface is a challenge.

If you could create an automation that would find all the terms based on the source location (res.city) and automatically copy the same term from English to the new language, that would be great.

**Comment by francesco-ooops** [comment, marked as answer]
Have you tried from this page:
https://translation.odoo-community.org/projects/l10n-brazil-14-0/l10n-brazil-14-0-l10n_br_base/pt_BR/
and then clicking Files > Upload translation?

Francesco

Il giorno mer 11 ott 2023 alle ore 16:17 Antônio Neto <
***@***.***> ha scritto:

> Does anyone know if there is any way to edit in mass or perform an
> automated action on the weblate?
>
> For example, in Brazil we have registered 5 thousand cities
> <https://github.com/OCA/l10n-brazil/blob/14.0/l10n_br_base/data/res.city.csv>
> and the name of these cities does not require translation, but they are
> pending in the weblate, adjusting them one by one manually through the
> weblate's web interface is a challenge.
>
> If you could create an automation that would find all the terms based on
> the source location (res.city) and automatically copy the same term from
> English to the new language, that would be great.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/OCA/discussions/118>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANHLKUMGXOPH3YCAOZM56P3X62TAFANCNFSM6AAAAAA54CTCCU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 

*Francesco Foresti*
*Sicurpharma Srl*
***@***.***
+39 333 8123 790

**Reply by antoniospneto** [reply]
thanks! I didn't see that option

**Reply by pedrobaeza** [reply]
Does it make sense to have translatable all that information? Maybe it's better to change the model definition to not be translatable.

**Reply by antoniospneto** [reply]
Indeed, you are right. Copying the term doesn't seem the most correct to me, as it is still pending verification (failing checks). In any case, it is necessary to go term by term to dismiss it. I also haven't found a way to dismiss the terms in mass.
![image](https://github.com/OCA/odoo-community.org/assets/634278/2e30cb3c-03cd-45da-b940-21eeb95fcea4)
![image](https://github.com/OCA/odoo-community.org/assets/634278/953d2e0c-6d5c-4315-b123-1255472e6116)

I'm exploring the tool to try to edit the metadata in mass, to mark it as 'not be translatable,' but it seems not to be active for my use
